includes/components/class-attribute.php

```diff
@@ -1513,6 +1513,10 @@ public function add_meta_boxes( $meta_boxes ) {
 		foreach ( $this->get_models() as $model ) {
 			foreach ( $meta_box_args as $meta_box_name => $meta_box ) {
 
+				if ( 'attribute_search' === $meta_box_name && ( ! hp\get_array_value( hivepress()->get_config( 'post_types' ), $model ) || ! hp\get_array_value( hp\get_array_value( hivepress()->get_config( 'post_types' ), $model, [] ), 'has_archive' ) ) ) {
```

Comment:
This can be moved lower, e.g. after line 1536, since the condition below filters attribute-related meta boxes only. The check can be made in the same way, it's ok to check the user model directly:

if ( 'listing' === $model || ...

You can also get the post type variable before the condition for readability